src/main/resources/db/migration/V6.2.0.2__8739-publisher-during-harvesting.sql

```diff
@@ -0,0 +1 @@
+update foreignmetadatafieldmapping set datasetfieldname = 'distributorName' where foreignfieldxpath = ':publisher';
```

Comment:
What about datasets that are already harvested? The docs say, "This change affects all newly harvested datasets."

The plan is to have a mix?

Or should we add to the release note that all datasets should be re-harvested? Is that crazy?

Reply:
Indeed, the fix isn't retroactive, unless all datasets are re-harvested. This is not great but I have no idea how we could do this another way.

Maybe we could only suggest re-harvesting only if people are affected by/care about the discrepancy it fixes?

Reply:
@plecor for now, can you please add something to the release note. "All datasets should be re-harvested to pick up this change." Or whatever makes sense to you. Thanks.

Reply:
Ok. To reflect this, the full text of the release note is now:

> The publisher value of harvested datasets is now attributed to the dataset's distributor instead of its producer. This improves the citation associated with these datasets, but the change only affects newly harvested datasets. All datasets should be re-harvested if you wish to pick up this change on already harvested datasets. For more information, see [the guides](https://dataverse-guide--9013.org.readthedocs.build/en/9013/admin/harvestclients.html#harvesting-client-changelog), #8739, and #9013.